app/policies/idv/flow_policy.rb

```diff
@@ -0,0 +1,158 @@
+module Idv
+  class FlowPolicy
+    include Rails.application.routes.url_helpers
+
+    NEXT_STEPS = Hash.new([])
+    NEXT_STEPS.merge!(
+      {
+        root: [:welcome],
+        welcome: [:agreement],
+        agreement: [:hybrid_handoff, :document_capture],
+        hybrid_handoff: [:link_sent, :document_capture],
+        link_sent: [:ssn],
+        document_capture: [:ssn], # in person?
+        ssn: [:verify_info],
+        verify_info: [:phone],
+        phone: [:phone_enter_otp],
+        phone_enter_otp: [:review],
+        review: [:personal_key],
+        personal_key: [:success],
+      },
+    )
```

Comment:
I would consider making this one statement with a `.freeze` at the end
```suggestion
    NEXT_STEPS = Hash.new([]).merge(
      {
        root: [:welcome],
        welcome: [:agreement],
        agreement: [:hybrid_handoff, :document_capture],
        hybrid_handoff: [:link_sent, :document_capture],
        link_sent: [:ssn],
        document_capture: [:ssn], # in person?
        ssn: [:verify_info],
        verify_info: [:phone],
        phone: [:phone_enter_otp],
        phone_enter_otp: [:review],
        review: [:personal_key],
        personal_key: [:success],
      },
    ).freeze
```